sonic_installer/main.py

```diff
@@ -533,14 +533,14 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
             raise click.Abort()
     else:
         # Verify not installing non-secure image in a secure running image
-        if not bootloader.verify_secureboot_image(image_path) and not force:
+        if not force and not bootloader.verify_secureboot_image(image_path):
             echo_and_log("Image file '{}' is of a different type than running image.\n".format(url) +
                 "If you are sure you want to install this image, use -f|--force|--skip-secure-check.\n" +
                 "Aborting...", LOG_ERR)
             raise click.Abort()
 
         # Verify that the binary image is of the same platform type as running platform
```

Comment:
>platform [](http://example.com/codeflow?start=54&length=8)

Platform has multiple meanings in SONiC context. Suggest platform -> asic
running platform -> running platform's asic type

Reply:
will change